website-next/pages/blog.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react'
+import { Header } from '../components/header'
+import { Menu } from '../components/Menu'
+import { MainTitle } from '../components/main-title'
+import { Paragraph } from '../components/paragraph'
+import { GhostBrowser } from '../components/ghostbrowser'
+import { HostedImage } from '../components/hosted-image'
+
+export default function Blog() {
```

Comment:
This should probably live under something like `blog/introducing-utopia.tsx` so that any links to this don't break when we later add other blog posts

Reply:
Or alternatively the actual blog post should live there, and this should inline it

Reply:
yes. let's decide a nice permalink that we stick to.

I like `https://utopia.app/blog/introducing-utopia` as Rheese suggested!

Reply:
I moved it to `blog/introducing-utopia` and also /blog and the landing page loads this for now